package/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "nickel-lang-package"
+description = "The Nickel Package Manager (npm)"
+version = "0.1.0"
```

Comment:
It's  not very important for this PR, but I wonder if we shouldn't make it target either `nickel-lang-core` or the Nickel version (`nickel-lang-cli`) (probably the latter, since it does provide new CLI commands), to avoid the proliferation of uncorrelated versions. On the other hand, this removes the possibility of updating those crates independently.

Reply:
I think it makes some sense to have the version in sync with `nickel-lang-cli`. It does require a little bit of manual intervention to keep them in sync: I can put `version.workspace = true` in `package/Cargo.toml`, but I need to repeat the "1.9.0" in the workspace's dependencies list.

Reply:
Right, but none of the other non-"independent" crates are a dependency of other crates in the workspace. Naively adding `package` to the list of crates gives

```
 -- Updating cross-dependencies...
  -- Patching cross-dependency lsp-harness in ./Cargo.toml to version 0.1.0
  -- Patching cross-dependency nickel-lang-core in ./Cargo.toml to version 0.11.0
  -- Patching cross-dependency nickel-lang-package in ./Cargo.toml to version {
  "workspace": true
}
sed: -e expression #1, char 87: unterminated `s' command
++ Unexpected exit. Cleaning up...
```

(Also, I guess we should add `git` and `vector`)